src/frontend/docker/nginx.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM nginx:alpine
+
+LABEL author="Mohd. Shariq" 
+
+# Copy custom nginx config
+COPY ./nginx.conf /etc/nginx/nginx.conf
+
+EXPOSE 8080
+
+ENTRYPOINT ["nginx", "-g", "daemon off;"]
```

Comment:
daemon off? can we add a comment in file? and what is the other option? when to use what?

Reply:
nginx can be configured not to self-daemonize by passing `daemon off`.

*nginx doc says* 
```
Syntax:  daemon on | off
Default:  daemon on;
```
Determines whether nginx should become a daemon. Mainly used during development.